src/universal/modules/meeting/components/MeetingLobby/MeetingLobby.js

```diff
@@ -25,7 +25,7 @@ const MeetingLobby = (props) => {
   const {members, team, styles} = props;
   const {id: teamId, name: teamName} = team;
   const onStartMeetingClick = createStartMeetingHandler(members);
-  const shortUrl = makeHref(`/team/${teamId}`);
+  const shortUrl = makeHref(`/meeting/${teamId}/lobby`);
```

Comment:
I think `/meeting/${teamId}/` is sufficient as it will redirect to whatever active phase.

Reply:
Oh, nice! Will try this.